### ISSUE TITLE
Transfer AWS EIP Association to another AWS ENI

### ISSUE THREAD

**Issue by dyn0m1ght** [gruntwork-io/terragrunt#1463]
I have a terraform module with TG providing the inputs that creates an ENI and associates an EIP to it.  I am trying to destroy the ENI and keep the EIP while moving the EIP to a new EC2 instance.  

So basically I want to:

1. Destroy the ENI 
2. Keep the EIP
3. Reassociate the EIP to a different ENI

I'm sure someone has down this before or could point me in the right direction?

**Comment by yorinasub17** [comment]
You should be able to use [terraform resource targeting](https://www.hashicorp.com/blog/resource-targeting-in-terraform) to destroy just the ENI. Just know that terraform will destroy all the things that depend on the target too.

**Comment by dyn0m1ght** [comment]
What I was able to do was by doing a terraform state remove on the resource, it removes it from being tracked in the state file.
By doing this on the AWS EIP ENI, I'm able to manually move the resource to the new state file, update the MD5 hash in Dynamodb to match what is expected in the Terragrunt code, and update the code to match the state file.

I'm sure there are other ways to do this, but this is the way I accomplished it.
